src/hotspot/share/opto/superword.cpp

```diff
@@ -2943,6 +2945,24 @@ bool SuperWord::is_vector_use(Node* use, int u_idx) {
     }
     return true;
   }
+
+  if (VectorNode::is_vpopcntq(use)) {
+    // VPOPCNTQ takes longs and produces ints - hence the special checks
+    // on alignment and size.
```

Comment:
Use IR node reference instead of target specific instruction.

Reply:
Thanks Jatin for noticing that! Will rename the functions to have generic names instead of target specific names. Will also modify the comment to be generic...